README.md

```diff
@@ -24,22 +24,75 @@ Please have a look at [the milestones](https://github.com/kubernetes-incubator/e
 
 ## Features
 
-* External DNS should be able to create/update/delete records on Cloud Provider DNS server
-* Configurable Cloud Provider
+* External DNS should be able to create/update/delete records on multiple cloud providers
+* The used cloud provider should be configurable at runtime
 * External DNS should take the ownership of the records created by it
-* Support Kubernetes Service(type=Loadbalancer), Ingress and NodePorts
-* Support DNS naming via annotations
-* Fault Tolerance
+* It should support Kubernetes Services with `type=Loadbalancer` and `type=NodePort`, Ingresses and Nodes
+* Allow to customize external name via annotations
+* It should be fault tolerance to individual pod failures
 * Support weighted records annotations - allow different resources share same hostname, and respective weighted records should be created.
 * Support multiple hosted zones - therefore External DNS should be able to create records as long as there exist a hosted zone matching the desired hostname
 
-## Minor Features
+## Nice to have
 
 * Should do smart cloud provider updates, i.e. Cloud Provider API should be called only when necessary
 * High Availability - should be possible to run multiple instances of External DNS
 * Should be able to monitor resource changes via K8S API for quick updates
 * New DNS record sources (e.g. TPRs) and targets (e.g. Azure DNS) should be pluggable and easy to add
 
+## Example
+
+The following gives a rough view of a barely working `external-dns`. There are a couple of requirements to make the following work for you.
+* Have a local Go 1.7+ development environment.
+* Have access to a Google project with the DNS API enabled.
+* Have access to a Kubernetes cluster that supports exposing Services, e.g. GKE.
+* Have a properly setup, **unused** and **empty** hosted zone in Google.
+
+Build the binary.
+
+```console
+$ mkdir -p $GOPATH/src/github.com/kubernetes-incubator/external-dns
+$ cd $GOPATH/src/github.com/kubernetes-incubator/external-dns
+$ git clone https://github.com/kubernetes-incubator/external-dns.git .
+$ go build -o build/external-dns .
+```
+
+Run an application and expose it via a Kubernetes Service.
+
+```console
+$ kubectl run nginx --image=nginx --replicas=1 --port=80
+$ kubectl expose deployment nginx --port=80 --target-port=80 --type=LoadBalancer
+```
+
+Annotate the service with your desired external DNS name (change `example.org` to your domain).
+
+```console
+$ kubectl annotate service nginx "external-dns.alpha.kubernetes.io/hostname=nginx.example.org."
+```
+
+Run a single sync loop of `external-dns`. In a real setup this would run constantly in your cluster (change the Google project and zone identifier to the one you have).
```

Comment:
I would highlight that the testing google zone should be used, to avoid some unexpected outcome :D